### ISSUE TITLE
Black screen in the VR headset when PostProcessLayer is on the camera

### ISSUE THREAD

**Issue by demonixis** [Unity-Technologies/PostProcessing#811]
##### What happened?
Black screen in the game preview and in the VR headset when the `PostProcessLayer` is on the camera.

##### Post-processing stack version (v1, v2)?
V2.

##### Unity version, operating system, target platform (standalone windows, mac, iOS, PS4...)?
Unity 2019.x on Windows 10 targeting Windows.

**Comment by Timeswitch** [comment]
This might be the same issue we are having :(
https://github.com/Unity-Technologies/PostProcessing/issues/795

**Comment by olgv** [comment]
Same here, though I used it before with no problems.
And old bug that seems to circle back?!

**Comment by AlphaAnklav** [comment]
Same?
#813

**Comment by olgv** [comment]
@timeswitch this relates to "Single/Multi Pass" option on the Player Settings > XR Settings 

This helped me understand how it works:
https://docs.unity3d.com/Manual/SinglePassStereoRendering.html

**Comment by Timeswitch** [comment]
> @Timeswitch, this relates to "Single/Multi Pass" option on the Player Settings > XR Settings
> 
> This helped me understand how it works:
> https://docs.unity3d.com/Manual/SinglePassStereoRendering.html

Might be, but I get it with either option :(

**Comment by dim8** [comment]
Could this issue be related to the *"Directly to Camera Target"* (`finalBlitToCameraTarget`) option of `PostProcessLayer`? Unchecking it, removes the black screen, but leaves a lot of distortion artefacts in the camera view.

**Comment by tibor0991** [comment]
I confirm the issue with "Directly to Camera Target" on my Oculus Go with Unity 2019.1.3, it completely broke my post-process stack.
As a temporary measure I'm attempting to disable the whole post-process stack in order to make the app work but I need it back up ASAP.

PS: Please, stop breaking the Post-Process stack at each major update, it's getting on my nerves.

**Comment by capitaljmedia** [comment]
Yes, this is the case for me too. Having the directly to camera setting on prevents any rendering on the headset, having it off makes the image a bit blurry. Would be great to get a fix for this :)

**Comment by demonixis** [comment]
Hey, I finally opened an [issue here](https://fogbugz.unity3d.com/default.asp?1162238_f484l6c5jlu24ub9). I guess we're not so much developers to use Unity 2019 + VR + Post Processing because no fix with 5 minor versions.

**Comment by JashanChittesh** [comment]
Hrm ... two months and still no update?

**Comment by MrLeggo** [comment]
Same issues here on Quest.
